### PR TITLE
bugfix/duplicated-requests

### PR DESCRIPTION
**Pull request by pgs-gmartowski**

### CODE REVIEW

**Comment by james-simpson-ST** [comment]
lgtm - great job guys!

File: src/core/integrations/GoogleAnalytics.ts [hunk 1]
```diff
@@ -62,28 +61,33 @@ export class GoogleAnalytics {
...
     });
   }
 
-  private _insertGALibrary() {
-    this._gaLibrary = DomMethods.insertScript('head', GoogleAnalytics.GA_SCRIPT_SRC);
-    this._gaLibrary.async = true;
-    document.head.appendChild(this._gaLibrary);
+  private _insertGALibrary(): void {
+    DomMethods.insertScript('head', { async: 'async', src: GoogleAnalytics.GA_SCRIPT_SRC, id: 'googleAnalytics' }).then(
+      () => {
```
Comment (wirus15) [inline src/core/integrations/GoogleAnalytics.ts]: this `then` can be removed

Comment (pgs-gmartowski) [reply src/core/integrations/GoogleAnalytics.ts]: true :) we can ignore Promise here

Comment (pgs-gmartowski) [reply src/core/integrations/GoogleAnalytics.ts]: done

**Pull request merged**